projects/eslint-plugin/rules/portal/lib/rules/no-global-fetch.js

```diff
@@ -16,7 +16,7 @@ module.exports = {
 
 		const isValidDefaultImport = (node) => {
 			return (
-				node.source.value.endsWith('/fetch.es') &&
+				node.source.value.endsWith('/fetch') &&
```

Comment:
Instead of choosing one or the other, lets allow for both since it is likely we will have a point in DXP where both exist or it may get back ported to older versions that use `.es`
```suggestion
				(node.source.value.endsWith('/fetch') || node.source.value.endsWith('/fetch.es')) &&
```